<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(34, 34, 46);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>561</width>
      <height>751</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::PreventContextMenu</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #22222e;
border: 2px solid rgb(255, 0, 255);
border-radius: 10;
color: white</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>541</width>
       <height>71</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 0, 255, 255), stop:1 rgba(0, 162, 255, 255));
border-radius: 10;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>541</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Trajan</family>
        <pointsize>20</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">border: px solid #f66867;
border-radius: 10;</string>
      </property>
      <property name="text">
       <string>Dota Builds Generator</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>541</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #22222e;
border: 2px solid rgb(255, 0, 255);
border-radius: 10;
color: white</string>
     </property>
     <property name="locale">
      <locale language="Russian" country="Russia"/>
     </property>
     <property name="text">
      <string>Привет, пора посоздавать пару крутых сборок :) 
Напиши, на какого героя хочешь сборку(На английском,
 без лишних симболов, например &quot;Shadow Fiend&quot;)</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>286</x>
       <y>210</y>
       <width>241</width>
       <height>45</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Trajan</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="tabletTracking">
      <bool>false</bool>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 0, 255, 255), stop:1 rgba(0, 162, 255, 255));
border: 0px solid #f66867;
border-radius: 10;
color: white</string>
     </property>
     <property name="text">
      <string>СГЕНЕРИРОВАТЬ</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="name1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>360</y>
       <width>499</width>
       <height>41</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>800</width>
       <height>41</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #22222e;
border: 2px solid rgb(1, 162, 255);
border-radius: 10;
color: white</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="name3">
     <property name="geometry">
      <rect>
       <x>238</x>
       <y>490</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>800</width>
       <height>41</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #22222e;
border: 2px solid rgb(1, 162, 255);
border-radius: 10;
color: white</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="name4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>620</y>
       <width>499</width>
       <height>41</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>800</width>
       <height>41</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #22222e;
border: 2px solid rgb(1, 162, 255);
border-radius: 10;
color: white</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="name2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>490</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>800</width>
       <height>41</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #22222e;
border: 2px solid rgb(1, 162, 255);
border-radius: 10;
color: white</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>211</y>
       <width>241</width>
       <height>45</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Вот сюда)</string>
     </property>
    </widget>
    <widget class="QLabel" name="icon1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>420</y>
       <width>75</width>
       <height>55</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #22222e;
border: 2px solid rgb(1, 162, 255);
border-radius: 0;
color: white</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="icon2">
     <property name="geometry">
      <rect>
       <x>134</x>
       <y>420</y>
       <width>75</width>
       <height>55</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #22222e;
border: 2px solid rgb(1, 162, 255);
border-radius: 0;
color: white</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="icon3">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>420</y>
       <width>75</width>
       <height>55</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #22222e;
border: 2px solid rgb(1, 162, 255);
border-radius: 0;
color: white</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="icon4">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>420</y>
       <width>75</width>
       <height>55</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #22222e;
border: 2px solid rgb(1, 162, 255);
border-radius: 0;
color: white</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="icon5">
     <property name="geometry">
      <rect>
       <x>454</x>
       <y>420</y>
       <width>75</width>
       <height>55</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #22222e;
border: 2px solid rgb(1, 162, 255);
border-radius: 0;
color: white</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="icon11">
     <property name="geometry">
      <rect>
       <x>134</x>
       <y>680</y>
       <width>75</width>
       <height>55</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #22222e;
border: 2px solid rgb(1, 162, 255);
border-radius: 0;
color: white</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="icon14">
     <property name="geometry">
      <rect>
       <x>454</x>
       <y>680</y>
       <width>75</width>
       <height>55</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #22222e;
border: 2px solid rgb(1, 162, 255);
border-radius: 0;
color: white</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="icon10">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>680</y>
       <width>75</width>
       <height>55</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #22222e;
border: 2px solid rgb(1, 162, 255);
border-radius: 0;
color: white</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="icon12">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>680</y>
       <width>75</width>
       <height>55</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #22222e;
border: 2px solid rgb(1, 162, 255);
border-radius: 0;
color: white</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="icon13">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>680</y>
       <width>75</width>
       <height>55</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #22222e;
border: 2px solid rgb(1, 162, 255);
border-radius: 0;
color: white</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="icon8">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>550</y>
       <width>75</width>
       <height>55</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #22222e;
border: 2px solid rgb(1, 162, 255);
border-radius: 0;
color: white</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="icon9">
     <property name="geometry">
      <rect>
       <x>454</x>
       <y>550</y>
       <width>75</width>
       <height>55</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #22222e;
border: 2px solid rgb(1, 162, 255);
border-radius: 0;
color: white</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="icon7">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>550</y>
       <width>75</width>
       <height>55</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #22222e;
border: 2px solid rgb(1, 162, 255);
border-radius: 0;
color: white</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="icon6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>550</y>
       <width>75</width>
       <height>55</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #22222e;
border: 2px solid rgb(1, 162, 255);
border-radius: 0;
color: white</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="hero">
     <property name="geometry">
      <rect>
       <x>214</x>
       <y>270</y>
       <width>128</width>
       <height>72</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 0;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>772</y>
      <width>75</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #22222e;
border: 0px solid rgb(255, 0, 255);
border-radius: 0;
color: rgb(82, 82, 111);</string>
    </property>
    <property name="text">
     <string>ⓒB3tMy3x</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../../Downloads/30588.jpg"/>
  <include location="30588.png"/>
 </resources>
 <connections/>
</ui>
